ts/src/binance.ts

```diff
@@ -8548,7 +8456,7 @@ export default class binance extends Exchange {
         if (internalInteger !== undefined) {
             internal = internalInteger ? true : false;
         }
-        const network = this.safeString (transaction, 'network');
+        const network = this.networkIdToCode (this.safeString (transaction, 'network'));
```

Comment:
this is a "breaking change" for users who rely on 'incorrect' value, but we eventually should move to the correct approach.